src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/Ncwms.java

```diff
@@ -112,8 +112,9 @@ private void proxyWmsRequest(HttpServletRequest request, HttpServletResponse res
             if (isCollectionWithTimeMismatch(layerDescriptor.geoserverName())) {
                 wmsParameters.remove("TIME");
             }
-
+            wmsParameters.remove("version");
             wmsParameters.put("VERSION", new String[] { wmsVersion });
```

Comment:
Just curious why we are removing the "version" and putting "VERSION". Maybe we can add a comment here so it is easy to follow.